tests/cases/standalone/alter/rename_table.sql

```diff
@@ -1,3 +1,7 @@
+CREATE SCHEMA test_rename_table;
```

Comment:
Why always create a new schema for these tests? I think testing `use` in `catalog` is enough.